psf_proposal.md

```diff
@@ -0,0 +1,88 @@
+
+  - Grant Type *: Development work
+
+  - Grant Title *: OSL Cookiecutter
+  Example: The fifth annual PyCon Atlantis
+
+- FAQ & Grants Program Info *:
+
+    I have read and understand these pages (es una casilla de selección)
+    Note: Before completing this application, read the web pages pertaining to the PSF’s grants program, freely available tools to run virtual meetings, and understand how funding works:
+    https://www.python.org/psf/grants
+    https://www.python.org/psf/grants/free-resources/
+
+Date *: 
+MM/DD/YYYY
+Provide the start date for your event/project
+Event’s/project’s website *:
+
+Requested Grant Amount (USD) *: 
+
+Grant Amount Breakdown *:
+
+What amount are you requesting from the PSF? If the grant is funding several components of a project/event, please provide a breakdown of what the funding will be used for. Example: Our event is requesting $1500 total ($1000 for captioning/transcripts, $500 for improved internet bandwidth for speakers).
+
+Budget *:
+Please include a link to a budget for the event or project work. This should show us an overview of all expenses and expected revenues. If you need to submit the budget via a document, please note that here and attach the document once you get to the "Other" section of this application.
+Objective *: 
+
+- What is the objective of your event/project?
+
+  Provide an up-to-date template with the best tools for creating Python packages.
+
+
+- Beneficiaries *:
+    Provide a description of the beneficiaries of this grant. If this request is for a sprint or virtual event, please tell us how many people are expected to attend. (e.g. if the grant is for a Python User Group (PUG) event, state the size of the group, the anticipated size of the event, and who in particular from the PUG benefits from the grant).
+    Regional Support
+    If you have funds remaining after your [event/development/etc], how do you plan to use them? For example: will your organization use remaining funding to help support regional Python efforts? Will remaining funds be saved for the next event? If so, please tell us about it!
+
+- Link to Code of Conduct *: https://opensciencelabs.org/guidelines/mentoring/en/coc/ 
+
+    Provide a link to your code of conduct. We require that your code of conduct be listed on the event website as well. Meetup.com Requests: a link to, or the full text of your CoC must be in the "About" section of your Meetup page, or we cannot send you an invite to the PSF Meetup Group
+
+- Percentage of Python *: 100%
+    What percentage of your event/project will concentrate on Python?
+
+- Follow-up activities *:
+    Provide a description of the planned grant follow-up activities. If this grant is approved, we will send you a link to our report form. In addition to that, we want to know how post event/project information will be shared with the community. This could be blog postings, post-event write-ups, web pages, project updates, where recordings will be posted, etc.
```

Comment:
As a follow up activity, we plan to have a blog post and a webinar about all updates done via this grant.